src/test/java/juja/microservices/keepers/service/KeepersServiceTest.java

```diff
@@ -114,4 +115,39 @@ public void addKeeper(){
         //Then
         assertEquals(expected, result);
     }
+    @Test
+    public void getActiveKeepers(){
+        //Given
+        Map<String, List<String>> expected = new HashMap<>();
+        expected.put("uuidTo2", Arrays.asList("sqlcmd"));
+        expected.put("uuidTo1", Arrays.asList("teems","sqlcmd"));
+
+        List<Keeper> listActiveKeepers = new ArrayList<>();
+        Keeper activeKeeper1 = new Keeper("uuidFrom1", "uuidTo1", "teems", new Date());
+        Keeper activeKeeper2 = new Keeper("uuidFrom2", "uuidTo1", "sqlcmd", new Date());
+        Keeper activeKeeper3 = new Keeper("uuidFrom1", "uuidTo2", "sqlcmd", new Date());
+        listActiveKeepers.add(activeKeeper1);
+        listActiveKeepers.add(activeKeeper2);
+        listActiveKeepers.add(activeKeeper3);
+
+        //When
+        when(repository.getActiveKeepers()).thenReturn(listActiveKeepers);
+
+        Map<String, List<String>> mapActiveKeepers = new HashMap(){};
+        for (Keeper keeper : listActiveKeepers) {
+            List<String> directions = new ArrayList<>();
+            if(mapActiveKeepers.containsKey(keeper.getUuid())) {
+                directions = mapActiveKeepers.get(keeper.getUuid());
+                directions.add(keeper.getDirection());
+                mapActiveKeepers.replace(keeper.getUuid(),directions);
+            }else{
+                directions.add(keeper.getDirection());
+                mapActiveKeepers.put(keeper.getUuid(), directions);
+            }
+        }
```

Comment:
please don't repeat the business logic from original service. You have to create expected result ( you did it), the parameter for the service method getActiveKeepers() - you also did it. And then just invoke this method from service. Received result should be compared with expected (assertEquals)